### PR TITLE
View Single Shoutout

### PR DESCRIPTION
**Pull request by ropeks**
Motivation
- So that user could view a single shoutout if logged in

Finish Line
- A single shoutout page that matches this design ----> https://www.figma.com/file/SbSjn8oAUtdWY1slSJeP9u/Bravo?node-id=247%3A2

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# “command” is your build command.
+# “publish” is the directory to publish (relative to the root of your repo).
```
Comment (samarv) [inline netlify.toml]: @nobioma1  any idea why am I seeing the whole file in didd

Comment (nobioma1) [reply netlify.toml]: @samarv The changes are from the "Setup Add App to Slack button" which @ropeks pulled into this branch before creating the view for single shoutouts

Comment (samarv) [reply netlify.toml]: @nobioma1  is it the same as the one in `develop`. do we need to scrap this file?

Comment (nobioma1) [reply netlify.toml]: @samarv the `netlify.toml` issue has been fixed by pulling the updated `develop` branch and merging it to this branch by @ropeks

**Pull request merged**